overlay.nix

```diff
@@ -238,11 +238,7 @@ let
 
         wfoverlap = callPackage ./pkgs/apps/wfoverlap { };
 
-        xtb = callPackage ./pkgs/apps/xtb {
-          turbomole = null;
-          cefine = null;
-          orca = self.orca;
-        };
+        xtb = callPackage ./pkgs/apps/xtb { };
```

Comment:
This breaks `qchem.tests.xtb`, since the tests relies on orca.

Reply:
Fixed